media/js/firefox/new/trailhead/thanks.js

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This file seems to be a duplicate and not used/needed? (I'm not opposed to using a duplicate for the old template, if that might make help separate concerns)

Reply:
Oops, that was the intent. Guess I only got half way there.